eng/Packages.props

```
@@ -17,32 +17,17 @@
     <PackageReference Update="Shouldly" Version="3.0.0" />
     <PackageReference Update="SourceLink.Create.CommandLine" Version="2.1.2" />
     <PackageReference Update="System.CodeDom" Version="4.4.0" />
-    <PackageReference Update="System.Collections.Concurrent" Version="4.3.0" />
     <PackageReference Update="System.Collections.Immutable" Version="5.0.0" />
-    <PackageReference Update="System.Diagnostics.TraceSource" Version="4.0.0" />
-    <PackageReference Update="System.IO.Compression" Version="4.3.0" />
-    <PackageReference Update="System.IO.FileSystem.Primitives" Version="4.3.0" />
-    <PackageReference Update="System.Linq.Parallel" Version="4.0.1" />
     <PackageReference Update="System.Memory" Version="4.5.4" />
-    <PackageReference Update="System.Net.Http" Version="4.3.4" />
     <PackageReference Update="System.Reflection.Metadata" Version="1.6.0" />
-    <PackageReference Update="System.Reflection.TypeExtensions" Version="4.1.0" />
     <PackageReference Update="System.Resources.Extensions" Version="4.6.0" />
```

```suggestion
    <PackageReference Update="System.Resources.Extensions" Version="4.6.0" />
    <PackageReference Update="System.Runtime.CompilerServices.Unsafe" Version="5.0.0"/>
```

It seems I should not have deleted this one. Will merge if it passes CI this time.